pkgs/top-level/all-packages.nix

```diff
@@ -14780,6 +14780,8 @@ with pkgs;
 
   xfstests = callPackage ../tools/misc/xfstests { };
 
+  xppen = callPackage ../tools/misc/xppen { };
```

Comment:
This package might need to be moved to [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md)

Reply:
It's from [RFC 140](https://github.com/NixOS/rfcs/pull/140), which was merged a while back. The plan is to move all packages into the new hierarchy at some point, but currently it's moving step by step. The CI is failing right now because #275539 was merged recently.

There's also a [nixcon talk](https://www.youtube.com/watch?v=vEpjHNZYwkA) if you're interested in a slightly less formal version of the topic. It provides some interesting historical details as well.